livespark-showcase/livespark-webapp/pom.xml

```diff
@@ -43,6 +43,12 @@
   </properties>
 
   <dependencies>
+    <!-- drools-compiler has dependency on org.eclipse.jdt.core.compiler:ecj:jar:3.5.1:compile, see http://code.google.com/p/google-web-toolkit/issues/detail?id=4479 -->
```

Comment:
Is this comment still relevant? drools-compiler now depends on ECJ 4.4.3 and I am not sure the original applies anymore.